src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -275,7 +275,7 @@ public boolean isAlive(InetAddressAndPort ep)
         // it's worth being defensive here so minor bugs don't cause disproportionate
         // badness.  (See CASSANDRA-1463 for an example).
         if (epState == null)
-            logger.error("Unknown endpoint: " + ep, new IllegalArgumentException(""));
+            logger.error("Unknown endpoint: {}", ep);
```

Comment:
Why are you removing the exception?  Pretty sure this was here so we could get a stack trace when it happened.